src/fiatExchanges/normalizeQuotes.ts

```diff
@@ -0,0 +1,194 @@
+  SUPPORTED_FIAT_ACCOUNT_TYPES,
+} from 'src/fiatconnect/FiatDetailsScreen'
+import {
+  CICOFlow,
+  FetchProvidersOutput,
+  isSimplexQuote,
+  PaymentMethod,
+  SimplexQuote,
+} from 'src/fiatExchanges/utils'
+import i18n from 'src/i18n'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { navigateToURI } from 'src/utils/linking'
+
+export interface NormalizedQuote {
+  quote: QuoteInfo
+  provider: ProviderInfo
+}
+
+export interface QuoteInfo {
+  paymentMethod: PaymentMethod
+  fee: number | null
+  kycInfo: string | null
+  timeEstimation: string
+  onPress: () => void
+}
+
+export interface ProviderInfo {
+  name: string
+  logo: string
+  id: string
+}
+
+const strings = {
+  oneHour: i18n.t('selectProviderScreen.oneHour'),
+  numDays: i18n.t('selectProviderScreen.numDays'),
+  idRequired: i18n.t('selectProviderScreen.idRequired'),
+}
+
+function getSettlementEstimation(lower: string | undefined, upper: string | undefined) {
+  // TODO: Dynamically generate time estimation strings
+  //
+  return strings.numDays
+}
+
+export function normalizeQuotes(
+  flow: CICOFlow,
+  fiatConnectQuotes: FiatConnectQuote[] | undefined,
+  externalProviders: FetchProvidersOutput[] | undefined
+): NormalizedQuote[] {
+  return [
+    ...normalizeFiatConnectQuotes(flow, fiatConnectQuotes),
+    ...normalizeExternalProviders(flow, externalProviders),
+  ].sort(sortQuotesByFee)
+}
+
+export const sortQuotesByFee = (
+  { quote: quote1 }: NormalizedQuote,
+  { quote: quote2 }: NormalizedQuote
+) => {
+  const providerFee1 = quote1.fee ?? 0
+  const providerFee2 = quote2.fee ?? 0
+
+  return providerFee1 > providerFee2 ? 1 : -1
+}
+
+export function normalizeFiatConnectQuotes(
+  flow: CICOFlow,
+  quotes: FiatConnectQuote[] | undefined
+): NormalizedQuote[] {
+  const normalizedQuotes: NormalizedQuote[] = []
+
+  quotes?.forEach((quote) => {
+    // Iterate through every FiatAccountType. A single quote can have multiple
+    Object.entries(quote.fiatAccount).forEach(
+      ([key, value]: [string, FiatAccountTypeQuoteData | undefined]) => {
+        // Check if we support the FiatAccountType and at least one of the FiatAccountSchemas
+        // We choose the first FiatAccountSchema that is supported
+        const schema = value?.fiatAccountSchemas.find((schema) =>
+          SUPPORTED_FIAT_ACCOUNT_SCHEMAS.has(schema.fiatAccountSchema)
+        )
+        if (SUPPORTED_FIAT_ACCOUNT_TYPES.has(key as FiatAccountType) && schema) {
+          normalizedQuotes.push({
+            quote: {
+              paymentMethod: PaymentMethod.Bank,
+              fee: value?.fee !== undefined ? parseFloat(value.fee) : null,
+              kycInfo: quote.kyc.kycRequired ? strings.idRequired : null,
+              timeEstimation: getSettlementEstimation(
+                value?.settlementTimeLowerBound,
+                value?.settlementTimeUpperBound
+              ),
+              onPress: () => {
```

Comment:
I can imagine this onPress function gets long and complicated after we add kyc stuff. How would props like fiatAccountSchema passed from kyc screen to fiatDetailsScreen ? I was imagining to pass around the raw FiatConnectQuote around to all the following screens. 

the other thing to think about is how to unit test onPress. I'm thinking maybe having a  helper function that returns onPress would make it easier to test